api/app/signals/apps/dataset/management/commands/load_cbs_for_municipality.py

```diff
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+"""
+Extract neighborhood data for a given municipality from CBS data.
+
+Note:
+- Centraal Bureau voor de Statistiek (CBS) neighborhood and municipal map data
+  must be present in the Signalen database.
+- The Signalen instance that we run this script for must serve one municipality.
+- AreaTypes will be added that have names `<municipal_code>-wijk` and
+  `<municipal_code>-buurt`.
+- The Dutch "buurt" and "wijk" both translate to neighborhood, but in the
+  Netherlands "wijk" areas are larger and consist of "buurt" areas. We use the
+  Dutch terms below - the use of CBS data makes this code specific to the
+  Netherlands.
+"""
+from django.core.management import BaseCommand, CommandError
+from django.db import transaction
+
+from signals.apps.signals.models import Area, AreaType
+
+REQUIRED_DATASETS = {
+    # Dataset names must match those defined in signals.apps.dataset.sources.cbs
+    'CBS_MUNICIPAL_BORDERS_DATASET': 'cbs-gemeente-2021',
+    'CBS_WIJK_DATASET': 'cbs-wijk-2021',
+    'CBS_BUURT_DATASET': 'cbs-buurt-2021',
+}
+
+
+class Command(BaseCommand):
+    def _cbs_data_present(self):
+        """
+        We need the CBS data municipal borders and neighborhood borders
+        """
+        all_present = True
+        for value in REQUIRED_DATASETS.values():
+            if not AreaType.objects.filter(code=value).exists():
+                all_present = False
+                msg = f'Required area type {value} is not present in database!'
+                self.stdout.write(msg)
+                continue
+
+            area_type = AreaType.objects.get(code=value)
+            if not Area.objects.filter(_type=area_type).count():
+                all_present = False
+                msg = f'No area data associated with AreaType {value}!'
+                self.stdout.write(msg)
+
+        return all_present
+
+    def _municipal_code_is_correct(self, municipal_code):
+        """Municipal code must be present in CBS data"""
+        # Assumes that required dataset (municipal borders) is present.
+        area_type = AreaType.objects.get(code=REQUIRED_DATASETS['CBS_MUNICIPAL_BORDERS_DATASET'])
```

Comment:
Same as comment above. This could give you an error of there are more than one AreaType objects with the same code.

Reply:
The code has a `unique=True` in the model, resolving this comment.